docs/management/admin/endpoint-command-ref.asciidoc

```diff
@@ -0,0 +1,343 @@
+[[endpoint-command-ref]]
+= {elastic-endpoint} command reference
+
+This page lists the commands for management and troubleshooting of {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention.
+
+[NOTE]
+====
+* The service is not added to the `PATH` system variable, so you must prepend the commands with the full OS-dependent path:
```

Comment:
The previous sentence calls it a "component" and this calls it a "service." It's not majorly confusing, but just feels inconsistent. We could just call it by name here:
```suggestion
* {elastic-endpoint} is not added to the `PATH` system variable, so you must prepend the commands with the full OS-dependent path:
```